src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(double radius) {
+        setColor(Color.WHITE.name());
+        this.radius = radius;
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: circle, area: " + obtainTheArea() + " sq. units, radius: "
+                + radius + " units, color: " + getColor());
```

Comment:
The method name 'toDraw' does not follow Java naming conventions for methods. It should be a verb, such as 'draw'. Additionally, it seems you have placed the drawing logic in a method that should only calculate area. Consider separating concerns by having a dedicated method for drawing.